packages/whats-new/src/whats-new-page.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { GuidePage } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+
+function WhatsNewPage( {
+	pageNumber,
+	isLastPage,
+	alignBottom = false,
+	heading,
+	description,
+	imageSrc,
+	link,
+} ) {
+	const __ = useI18n().__;
+
+	useEffect( () => {
+		recordTracksEvent( 'calypso_block_editor_whats_new_slide_view', {
+			slide_number: pageNumber,
+			is_last_slide: isLastPage,
+		} );
+	}, [ isLastPage, pageNumber ] );
+
+	return (
+		<GuidePage className="whats-new-page__container">
+			<div className="whats-new-page__text">
+				<h1 className="whats-new-page__heading">{ heading }</h1>
```

Comment:
Any chance heading might not exist? Should we check before we render the `<h1>`?